.packit.yaml

```
@@ -38,3 +38,20 @@ jobs:
       # do not get the version from a tag (git describe) but from the spec file
       get-current-version:
         - grep -oP '^Version:\s+\K\S+' packaging/command-line-assistant.spec
+  - job: tests
+    trigger: pull_request
+    fmf_url: "https://gitlab.cee.redhat.com/rhel-lightspeed/enhanced-shell/cla-tests"
+    fmf_ref: main
+    use_internal_tf: True
+    tmt_plan: "plans"
+    targets:
+      epel-9-x86_64:
+        distros: [RHEL-9-Nightly]
+      epel-10-x86_64:
+        distros: [RHEL-10-Nightly]
+    tf_extra_params:
+      environments:
+        - settings:
+            provisioning:
+              tags:
+                BusinessUnit: sst_conversions
```

```suggestion
    tf_extra_params:
      environments:
        - arch: x86_64
        - os:
            compose: ""
        - settings:
            provisioning:
              tags:
                BusinessUnit: sst_conversions
```
We can try this to force the container.